test/unit/network_spec.js

```diff
@@ -67,11 +67,15 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
     // 'moz-chunked-array' response type.
     // TODO enable for other browsers, e.g. when fetch/streams API is supported.
```

Comment:
As mentioned in https://github.com/mozilla/pdf.js/pull/9050/files/#r154503634, these comments are no longer necessary.